ffmpeg_normalize/_streams.py

```diff
@@ -155,7 +155,7 @@ def parse_loudnorm_stats(self):
             opts['dual_mono'] = 'true'
 
         filter_str = '[0:{}]'.format(self.stream_id) + \
-            'loudnorm=' + dict_to_filter_opts(opts)
+            self.ffmpeg_normalize.pre_filter + ',' + 'loudnorm=' + dict_to_filter_opts(opts)
```

Comment:
This will break if `pre_filter` is not set, as it will be empty.

Instead, the actual filters should be combined by creating a list, adding the pre-filter only if it exists, and then, then joining the filter list with `,`.